fedimintd/src/fedimintd.rs

```diff
@@ -30,17 +30,22 @@ pub struct ServerOpts {
     /// Path to folder containing federation config files
     pub data_dir: PathBuf,
     /// Password to encrypt sensitive config files
+    // TODO: should probably never send password to the server directly, rather send the hash via
+    // the API
     #[arg(env = "FM_PASSWORD")]
     pub password: String,
     /// Port to run admin UI on
     #[arg(long = "listen-ui", env = "FM_LISTEN_UI")]
     pub listen_ui: Option<SocketAddr>,
+    /// After an upgrade the epoch must be passed in
```

Comment:
Unrelated question to the PR but since I'm thinking of it: if you were a guardian that was not apart of coordinating the shutdown (maybe you were offline), but you wanted to re-join after the upgrade occurred, how would you figure out this number?

Reply:
Ideally users wouldn't be calling this from the command-line but via an API call from an admin UI.

Still a lot of work to do on that front.